firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionHeaderView.swift

```diff
@@ -0,0 +1,99 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import UIKit
+import Common
+import Shared
+import ComponentLibrary
+
+class TrackingProtectionHeaderView: UIView {
+    let siteTitleLabel: UILabel = .build { label in
+        label.adjustsFontForContentSizeCategory = true
+        label.textAlignment = .center
+        label.font = FXFontStyles.Regular.headline.scaledFont()
+        label.numberOfLines = 2
+        label.accessibilityTraits.insert(.header)
+    }
+
+    private var closeButton: CloseButton = .build { button in
+        button.layer.cornerRadius = 0.5 * TPMenuUX.UX.closeButtonSize
+    }
+
+    var backButton: UIButton = .build { button in
+        button.layer.cornerRadius = 0.5 * TPMenuUX.UX.closeButtonSize
+        button.clipsToBounds = true
+        button.setTitle(.KeyboardShortcuts.Back, for: .normal)
+        button.setImage(UIImage(imageLiteralResourceName: StandardImageIdentifiers.Large.chevronLeft)
+            .withRenderingMode(.alwaysTemplate),
+                        for: .normal)
+        button.titleLabel?.font = TPMenuUX.Fonts.viewTitleLabels.scaledFont()
```

Comment:
Future fonts should be using `FXFontStyles`

Reply:
`viewTitleLabels: TextStyling = FXFontStyles.Regular.body` is already using it